packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/content/WpfClassLibrary-CSharp/.template.config/template.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "http://json.schemastore.org/template",
   "author": "Microsoft",
   "classifications": ["Common", "WPF"],
-  "name": "WPF Class library",
+  "name": "WPF Class Library",
   "generatorVersions": "[1.0.0.0-*)",
   "description": "A project for creating a class library that targets a .NET Core WPF Application",
```

Comment:
Should we replace `a .NET WPF` to `a .NET Core WPF` ?

We will update to dotnet 5

Reply:
Sounds reasonable, but I am not sure. @ryalanms, should we?

Reply:
Hi @AR-May.  I agree with @lindexi that .NET Core should be updated to .NET.  I will merge your typo fix for now, and either of you can create another PR for any additional updates.  Thanks.